teambit.bvm/link/link.ts

```diff
@@ -109,17 +110,19 @@ export async function linkOne(linkName: string, version: string | undefined, opt
     force: true,
   }
   const rawGeneratedLinks = binLinks.getPaths(binOpts);
-  await cmdShim(path.join(versionDir, source), rawGeneratedLinks[0], {
+  const generatedLink = {
+    source: versionDir,
+    target: rawGeneratedLinks[0]
+  }
+  const currentDir = path.join(config.getBvmDirectory(), linkName)
```

Comment:
maybe change this to be under links folder?
`path.join(config.getBvmDirectory(), 'links', linkName)`

Reply:
I don't know. It will make searching for the base path a bit harder. Also, in most of the cases there will be only a single symlink. `links` is also not a descriptive name. `active_versions` could be more descriptive.